roles/kubernetes/node/handlers/main.yml

```diff
@@ -9,7 +9,7 @@
   systemd:
     daemon_reload: true
 
-- name: Kubelet | reload kubelet
+- name: Kubelet | restart kubelet
```

Comment:
You should also change the `Kubelet | reload kubelet` (line 5) to match your change.

Reply:
This is not resolved you should change reload -> restart (on line 6 now)

Reply:
You should also rebase master :)